Library/Homebrew/cask/test/test_helper.rb

```diff
@@ -50,7 +50,7 @@ class TestCask < Cask; end
 end
 
 # pretend that the caskroom/cask Tap is installed
-FileUtils.ln_s HOMEBREW_LIBRARY.join("Taps", "caskroom", "homebrew-cask"), Tap.fetch("caskroom", "cask").path
+FileUtils.ln_s Pathname.new(ENV["HOMEBREW_LIBRARY"]).join("Taps", "caskroom", "homebrew-cask"), Tap.fetch("caskroom", "cask").path
```

Comment:
This feels like the wrong approach. Are we using the `HOMEBREW_LIBRARY` environment variable instead of the constant somewhere in the cask code?